Cargo.toml

```diff
@@ -1,13 +1,14 @@
 [package]
 edition = "2021"
 name = "holochain_scaffolding_cli"
-version = "0.1.7"
+version = "0.1.8"
 description = "CLI to easily generate and modify holochain apps"
 license-file = "LICENSE_CAL-1.0"
 homepage = "https://developer.holochain.org"
 documentation = "https://docs.rs/holochain_scaffolding_cli"
 repository = "https://github.com/holochain/scaffolding"
 readme = "README.md"
+resolver = "2"
```

Comment:
Technically unecessary because `edition = "2021"` uses this by default. You only have to set it in a workspace if you want this to apply from the top level. Won't do any harm though